### PR TITLE
Add player class and tests

### PR DESCRIPTION
**Pull request by Sillhouette**
Adds player class and tests for the following:
* Initialize method that sets the player name and default token
* set_token method that allows a player token to be changed

### CODE REVIEW

File: src/player.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Player():
+    def __init__(self, name, token="X"):
+        self.name = name
```
Comment (IanDCarroll) [inline src/player.py:3]: Does the player need a name just yet? Or is this more of a type? like "human" or "computer"?

File: test/player_test.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from src.player import Player
+
+class PlayerTest(unittest.TestCase):
+    def test_player_name(self):
+        name = "Chuck Norris"
```
Comment (IanDCarroll) [inline test/player_test.py:7]: ^ Freakin' Epic!

File: test/player_test.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        token = "X"
+
+        player = Player(name)
+
+        self.assertEqual(player.token, token)
+
+    def test_setting_player_token(self):
+        name = "Chuck Norris"
+        token = "¤"
```
Comment (IanDCarroll) [inline test/player_test.py:23]: "Chuck Norris only plays Tic Tac Toe with ninja stars."

**Review by IanDCarroll — COMMENTED** [review]
Some things to consider :) 
I like how simple it is and yet being separated by single responsibilities.

File: src/player.py (hunk 1 above)
Comment (Sillhouette) [reply src/player.py:3]: Ah you're right, the player doesn't need a name yet as we adjusted the story for this one. I had remnants of the old story in my head still and that's why I added the name functionality. Mostly to prepare for the future. 

Again, might have jumped the gun on it.

File: test/player_test.py (hunk 3 above)
Comment (dcmoore) [reply test/player_test.py:23]: Yeah, one could make the argument these should be the default values, lol

**Pull request merged**